docs/data/sdks/typescript-browser/migration.md

```diff
@@ -0,0 +1,497 @@
+---
+title: Migration Guide
+description: This is a guide on migrating from amplitude-js
+---
+
+Amplitude has released a new version of Browser SDK (`@amplitude/analytics-browser`), which features a plugin architecture, built-in type definition and broader support for front-end frameworks. The new version is not backwards compatible with `amplitude-js`. 
+
+To migrate to `@amplitude/analytics-browser`, the dependency and instrumentation must be updated.
+
+### Terminology
+
+* `amplitude-js`: Legacy Browser SDK
+* `@amplitude/analytics-browser`: New Browser SDK
+
+## Dependency
+
+For snippet installation, update your project"s snippet loader. The new snippet loader can be found here: [https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-browser#using-script-loader](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-browser#using-script-loader)
+
+For Node projects, update your dependency list in package.json.
+
+=== "amplitude-js"
+  ```json
+  {
+    "dependencies": {
+      "amplitude-js": "8"
+    }
+  }
+  ```
+
+=== "amplitude-js"
+  ```json
+  {
+    "dependencies": {
+      "@amplitude/analytics-browser": "1"
```

Comment:
I think we should promote semantic versioning e.g. "^1.0.0"

Reply:
```suggestion
      "@amplitude/analytics-browser": "^1.0.0"
```